openhtf/__init__.py

```diff
@@ -428,6 +428,14 @@ def wrap_or_copy(cls, func, **options):
     retval.options.update(**options)
     return retval
 
+  def _asdict(self):
```

Comment:
don't we have something for mutablerecords already? If the only reason for all of this is to ignore the non-serializable cls key, I think we should just add a `if isinstance(obj, type)` and skip all class definitions in `convert_to_base_types`